test/test.ts

```diff
@@ -4,6 +4,7 @@ const Dai = artifacts.require('Dai')
 const DaiHrd = artifacts.require('DaiHrd')
 
 const MAX_APPROVAL = bigintToHexString(2n**256n-1n)
+const DUST = 1n * 10n**18n; // Race with balanceOf and deposit
```

Comment:
That is a giant dust particle.  This should have a units, since there is also rontodai DUST.